Sources/MongoSwift/MongoCollection+FindAndModify.swift

```diff
@@ -198,6 +198,9 @@ public struct FindOneAndReplaceOptions: FindAndModifyOptionsConvertible, Decodab
     /// Specifies a collation to use.
     public var collation: BSONDocument?
 
+    /// A document or string that specifies the index to use to support the query.
+    public var hint: BSONDocument?
```

Comment:
We actually have a already `Hint` Swift type defined for similar use-cases elsewhere in the driver. Can this be updated to use that instead?

Reply:
(it's named `IndexHint`)